.gitignore

```diff
@@ -267,4 +267,5 @@ src/startupscriptgenerator/**/vendor/
 
 # Go debugging related generated files
 src/startupscriptgenerator/src/**/__debug_bin
-src/startupscriptgenerator/src/**/run.sh
\ No newline at end of file
+src/startupscriptgenerator/src/**/run.sh
+!**/dynamic/Dockerfile
```

Comment:
curious: what does ! stands for?

Reply:
We currently ignore all files which are dynamically generated in runtime folder (i.e the ones generated using `generateDockerfiles.sh`), but this also ignores the static Dockerfile created under `dynamic` folder, so this `!` says to not ignore that file and allow us to checkin